### PR TITLE
fix: BFF-651 Adjusted keyboard layout and added comma to as on iOS is not …

### PR DESCRIPTION
**Pull request by alexbejann**
…default for numbers

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/features/amount_presets/widgets/preset_form_field.dart [hunk 1]
```diff
@@ -56,7 +56,9 @@ class _PresetFormFieldState extends State<PresetFormField> {
         borderRadius: BorderRadius.circular(5),
         child: TextFormField(
           controller: widget.controller,
-          keyboardType: TextInputType.number,
+          keyboardType: const TextInputType.numberWithOptions(
```
Comment (pavelpavelko) [inline lib/features/amount_presets/widgets/preset_form_field.dart:59]: didn't know about that, nice)

File: lib/features/give/widgets/numeric_keyboard.dart [hunk 2]
```diff
@@ -135,70 +142,42 @@ class _NumericKeyboardState extends State<NumericKeyboard> {
...
+            color: widget.textColor,
           ),
-        ),
-      );
-
-  Widget _calcButton(String value) {
+        );
+    final size = MediaQuery.sizeOf(context);
+    var padding = 14.0;
+    if (size.height < 700) {
```
Comment (pavelpavelko) [inline lib/features/give/widgets/numeric_keyboard.dart:163]: what is 700?

**Review by pavelpavelko — APPROVED** [review]
looks good

File: lib/features/give/widgets/numeric_keyboard.dart (hunk 2 above)
Comment (alexbejann) [reply lib/features/give/widgets/numeric_keyboard.dart:163]: In this context is the screen height. If the screen height is less than 700 then we add that padding. I thought it was self-explanatory as I used the Media query of size

**Pull request merged**